[DOC] Update query editor content and screenshots (#3986)
grafana/tempo: docs/sources/tempo/configuration/_index.md

description: Learn about available options in Tempo and how to configure them.
weight: 400

> aliases:

Author: ```suggestion
```

description: Learn about available options in Tempo and how to configure them.
weight: 400
aliases:

> - ./ # /docs/tempo/<TEMPO_VERSION>/configuration/

Reviewer: You can leave this out, it's redirecting to itself.

```suggestion
```

            [max_live_traces: <uint64>]

            # Whether server spans should be filtered in or not.
            # If enabled, only parent spans or spans with the SpanKind of `server` will be retained
            [filter_server_spans: <bool> | default = true]

            # Number of blocks that are allowed to be processed concurently
            [concurrent_blocks: <uint> | default = 10]

            # A tuning factor that controls whether the trace-level timestamp columns are used in a metrics query.

>             # If a block overlaps the time window by less than this ratio, then we skip the columns.

Reviewer 2: Replace "then we skip" when "the columns are skipped".

Author: ```suggestion
            # If a block overlaps the time window by less than this ratio, then the columns are skipped.
```
